patch/llpcPatchInOutImportExport.cpp

```diff
@@ -3938,6 +3938,9 @@ void PatchInOutImportExport::PatchTcsBuiltInOutputExport(
                 LLPC_ASSERT(perPatchBuiltInOutLocMap.find(builtInId) != perPatchBuiltInOutLocMap.end());
                 uint32_t loc = perPatchBuiltInOutLocMap[builtInId];
 
+                llvm::Instruction *pInsertEndPos = pInsertPos->getParent()->getTerminator();
+                LLPC_ASSERT(pInsertEndPos);
```

Comment:
-> LLPC_ASSERT(pInsertAtEnd != nullptr);

Reply:
I don't think we need this assert. A basic block without a terminator is badly formed and would have asserted before now, right?

Reply:
OK, removing the assert.